<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>ZoMbiE</author>
 <class>Gui</class>
 <widget class="QWidget" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource_designer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pb_minus">
     <property name="text">
      <string>minus</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/minus.png</normaloff>:/minus.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sb_show"/>
   </item>
   <item>
    <widget class="QPushButton" name="pb_plus">
     <property name="text">
      <string>plus</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/plus.png</normaloff>:/plus.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
